src/authorization/WebAclReader.ts

```diff
@@ -29,10 +30,14 @@ const modesMap: Record<string, Readonly<(keyof AclPermission)[]>> = {
   [ACL.Control]: [ AclMode.control ],
 } as const;
 
-type AclSet = { targetAcl: Store; parentAcl?: Store };
+// Utility type for returning found ACL resources
+type AclMatch = { store: Store; identifier: ResourceIdentifier };
```

Comment:
Maybe just `AclDocument` or `AclStore`?
```suggestion
type AclDocument = { identifier: ResourceIdentifier, contents: Store };
```